pkg/apis/operator/v1/managementclusterconnection_types.go

```diff
@@ -40,6 +40,8 @@ type ManagementClusterConnection struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	Spec ManagementClusterConnectionSpec `json:"spec,omitempty"`
+
+	Status ManagementClusterConnectionStatus `json:"status,omitempty"`
```

Comment:
I can't really remember how this Status would be used, is the State value in this ever set?